spec/queries/get_duplicate_candidate_matches_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe GetDuplicateCandidateMatches do
+  let(:candidate1) { create(:candidate, id: '1', email_address: 'exemplar1@example.com') }
+  let(:candidate2) { create(:candidate, id: '2', email_address: 'exemplar2@example.com') }
+
+  before do
+    create(:application_form, candidate: candidate1, first_name: 'Jeffrey', last_name: 'Thompson', date_of_birth: '1998-08-08', postcode: 'w6 9bh ', submitted_at: Time.zone.now - 7.days)
+    create(:application_form, candidate: candidate2, first_name: 'Joffrey', last_name: 'Thompson', date_of_birth: '1998-08-08', postcode: 'w6 9bh ', submitted_at: Time.zone.now - 7.days)
```

Comment:
Maybe make the postcodes differ in case and trailing spaces, to test the trimming and lowering?